src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -50,16 +49,35 @@ class SensorFusion
 
    private:
     /**
-     * Updates relevant components of world based on a new data
+     * Updates refbox_stage and game_state based on new data
      *
-     * @param new data
+     * @param packet new SSL_Referee proto
      */
-    void updateWorld(const SSL_WrapperPacket &packet);
-    void updateWorld(const SSL_Referee &packet);
-    void updateWorld(
+    void updateRefboxStageAndGameState(const SSL_Referee &packet);
+
+    /**
+     * Updates timestamped_possession_state based on new data
+     *
+     * @pre ball, friendly_team, and enemy_team was updated
+     *
+     * @param tbots_robot_msgs New list of TbotsRobotMsg
+     */
+    void updatePossessionState(
         const google::protobuf::RepeatedPtrField<TbotsRobotMsg> &tbots_robot_msgs);
```

Comment:
Definitely not in scope for this PR, but might be worth renaming `TbotsRobotMsg` to something like `TbotsRobotStatusMsg`

Reply:
doing that here: https://github.com/UBC-Thunderbots/Software/pull/1620